engine/system/win/sys_main.cpp

```diff
@@ -842,14 +842,16 @@ bool sys_main_c::Run(int argc, char** argv)
 
 		// Run frame loop
 		while (exitFlag == false) {
-			glfwPollEvents();
+			if (minimized)
+				glfwWaitEvents(); // while minimized, puts process to sleep until it receives an event

```

Comment:
Maybe `glfwWaitEventsTimeout` should also be configured as discussed on Discord to ensure that processing does not stall completely?
Future work could include `glfwPostEmptyEvent` in subscript `l_SubScriptFunc` and `l_SubScriptSub` to wake the main thread up when there's pending func and sub calls.